src/tabular-api-surface.ts

```diff
@@ -736,12 +759,30 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
         err = null;
       }
 
+      const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
       if (mutationErrorsByEntryIndex.size !== 0) {
-        const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
         callback(new PartialFailureError(mutationErrors, err));
         return;
       }
-
+      if (err && timeoutExceeded) {
+        callback(
+          new PartialFailureError(
```

Comment:
The test is expecting no error code so it seems like it is expecting a PartialFailureError because it is also expecting errors per row so that's why we use this object.

Reply:
The test expect deadline exceeded ? https://github.com/googleapis/cloud-bigtable-clients-test/blob/c78ca1867bbf862ce7d742a999b7d45ae5edc478/tests/mutaterows_test.go#L229

Reply:
The line you highlighted is the entry level error (it is expecting error codes at the entry level). At the RPC level it is [expecting ok status](https://github.com/googleapis/cloud-bigtable-clients-test/blob/c78ca1867bbf862ce7d742a999b7d45ae5edc478/tests/mutaterows_test.go#L226) with [no code](https://github.com/googleapis/cloud-bigtable-clients-test/blob/c72bf49142e8208b586d51b240f46860ae5d4b28/tests/test_workflow.go#L599).

Reply:
The test is being changed to expect DEADLINE_EXCEEDED errors now and the Java test proxy is being changed to ensure the client error gets passed back to the test runner.

https://github.com/googleapis/cloud-bigtable-clients-test/pull/224
https://github.com/googleapis/java-bigtable/pull/2456

In the next update we will have the client apply a DEADLINE_EXCEEDED grpc error so that this new check passes instead of the old check.